scripts/tags/exturl.js

```diff
@@ -0,0 +1,57 @@
+/* global hexo */
+// Usage: {% exturl text url "title" %}
+// Alias: {% extlink text url "title" %}
+
+'use strict';
+
+var util = require('hexo-util');
```

Comment:
@iissnan em, yes. It was there always.

And i think need to check NPM and/or nodeJS version about that. My version still 0.10:
```
$ npm version
{ http_parser: '1.2',
  node: '0.10.48',
  v8: '3.14.5.11',
  ares: '1.9.0-DEV',
  uv: '0.10.34',
  zlib: '1.2.3',
  modules: '11',
  openssl: '1.0.1e-fips',
  npm: '1.3.6',
  'hexo-site': '0.0.0' }
```